kotlin-js/src/jsMain/kotlin/js/disposable/AsyncDisposable.using.kt

```diff
@@ -0,0 +1,44 @@
+@file:Suppress(
+    "NON_ABSTRACT_MEMBER_OF_EXTERNAL_INTERFACE",
+)
+
+package js.disposable
+
+import js.symbol.Symbol
+import kotlin.contracts.InvocationKind
+import kotlin.contracts.contract
+
+@PublishedApi
+internal suspend inline fun <T : AsyncDisposable?, R> using(disposable: T, block: (T) -> R): R {
```

Comment:
Redundant `inline` here and in other functions

Reply:
Non-local returns do not work without inlines.